packages/dashboard-frontend/src/components/EditorSelector/Definition/index.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2018-2024 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { Form } from '@patternfly/react-core';
+import React from 'react';
+
+import { EditorDefinitionField } from '@/components/EditorSelector/Definition/DefinitionField';
+import { EditorImageField } from '@/components/EditorSelector/Definition/ImageField';
+
+export type Props = {
+  onChange: (editorId: string | undefined, editorImage: string | undefined) => void;
+};
+
+export type State = {
+  editorId?: string;
```

Comment:
It is not just the editor ID. It could be the link to a container editor definition, URL, or editor ID.

I propose to change:

```editorId``` -> ```editorDefinition```

WDYT?